src/components/general/Speakers.vue

```diff
@@ -102,25 +104,51 @@ export default {
 	position: relative;
 }
 
-.image {
+.image-wrapper {
 	position: relative;
 	width: calc(100% - var(--icon-width) - .5rem);
 	height: 100%;
+	overflow: hidden;
 }
-.image>img {
-	width: 100%;
-	height: 100%;
-	object-fit: cover;
-	border-radius: var(--tile-radius);
-}
-.image-background {
+.image-wrapper>.positioning {
 	position: absolute;
 	left: 0;
 	bottom: 0;
-	z-index: -1;
 	width: 100%;
 	height: 80%;
+}
+.image-background {
 	border-radius: var(--tile-radius);
+	z-index: -1;
+	transition: var(--bio-transition);
+}
+.image-speaker {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+	border-radius: var(--tile-radius);
+	transition: var(--bio-transition);
+}
+
+.bio {
```

Comment:
We are going to need to dynamically update the text size to make sure the bios fit for all viewports. On my laptop there are a few bios that overflow. Did you cut the bios down already? 
![image](https://user-images.githubusercontent.com/41529410/119239510-9138e900-bb17-11eb-8bb1-41d15725a61c.png)

But even still, practically every bio overflows on mobile.
![image](https://user-images.githubusercontent.com/41529410/119239519-a44bb900-bb17-11eb-9617-5feb3a7da41d.png)

You could do media queries, but I'd play around with setting the `font-size` proportional to `vw`. 

As well, it might be worth adding a safety net in case a bio does overflow. Something like `text-overflow: fade` to make sure that in case something goes wrong we don't look to bad anyway. This safety would probably have to consider the top of the title overlay and make sure the bio doesn't go past it.

Reply:
I cut the bios down so that they fit on mine, but I can change it. Was there a specific character limit and would that fit on all screens? Because if it does, I'll go back and make sure all the bios fit the character limit.

Reply:
You'll have to test at different widths to see if the character limit fits on all screens. Please add the `vw font-size` below a certain width :)